src/main/_docs/use-che-as-an-ide/ide-debug.md

```diff
@@ -71,3 +71,116 @@ It is also possible to add this command as a Dockerfile instruction for a custom
+
+PHP CLI scripts can be debugged by setting the `QUERY_STRING` environment variable when executing the PHP script. For example, to debug the `hello.php` script you should execute the following command in the Terminal:
+
+```sh
+QUERY_STRING="start_debug=1&debug_host=localhost&debug_port=10137" php hello.php
+```
+
+Let's dissect the value of the `QUERY_STRING`:
+
+- `start_debug=1` says the PHP engine that we want to trigger a debug session for this execution.
+- `debug_host=localhost` says that the Zend Debugger Client runs on localhost (on the same host where the PHP engine runs).
+- `debug_port=10137` says that the Zend Debugger Client listens on port 10137.
+
+For convenience the PHP and Zend stacks have the `debug php script` command. It will run the PHP script, which is currently opened in the editor, with the required `QUERY_STRING` preprended to the launch command. It is a handy way for easily debugging CLI script without the need to remember the exact `QUERY_STRING` variable.
+
+## Debugging PHP Web Apps
+
+Debugging web apps is done in a similar way. The value of the `QUERY_STRING` used for debugging CLI scripts must be added as a query string to the URL of the debugged web page. This can be done either manually or by using a browser toolbar/extension that does it automatically. Such browser extensions also make it easier to debug POST requests.
+
+### Using Query Params in the URL
+
+The `?start_debug=1&debug_host=localhost&debug_port=10137` query string must be added to the URL. For example, to debug the `http://localhost:32810/web-php-simple/index.php` web page you should request the following URL in the browser:
+
+```
+http://localhost:32810/web-php-simple/index.php?start_debug=1&debug_host=localhost&debug_port=10137
+```
+
+### Using the zDebug Extension for Chrome
+
+The [zDebug](https://chrome.google.com/webstore/detail/zdebug/gknbnafalimbhgkmichoadhmkaoingil) extension can be used for easier triggering of debug sessions from the Chrome browser. The [Zend Debugger Extension](https://chrome.google.com/webstore/detail/zend-debugger-extension/aonajadpeeaijblinaeohfdmbgdpibba) is another extension that does the same job.
+
+It is important to configure the Chrome extension properly before using it for debugging PHP apps running in a Che workspace:
+
+1. Set `Debug Host` to `localhost` or `127.0.0.1`.
+1. Set `Debug Port` to `10137`.
+1. Set `Debug Local Copy` to `No`.
+
+Note that it is not the browser that opens the debug session to the Zend Debugger Client. This is done by the PHP engine that runs in the Che workspace. The browser just tells the PHP engine to do so. So the above settings are for the PHP engine (the Zend Debugger module in particular). Thus the `Debug Host` must be set to `localhost` and not the public host of the docker container running the Che workspace.
+
+In the end the zDebug settings should look like this:
+
+![zdebug-settings.png]({{ base }}/docs/assets/imgs/zdebug-settings.png)
```

Comment:
Should change to `![zdebug-settings.pngg]({{ base }}{{site.links["zdebug-settings.png"]}})`